src/main/java/bio/terra/service/dataset/flight/delete/DeleteDatasetDeleteStorageAccountsStep.java

```diff
@@ -0,0 +1,50 @@
+package bio.terra.service.dataset.flight.delete;
+
+import bio.terra.model.CloudPlatform;
+import bio.terra.service.dataset.Dataset;
+import bio.terra.service.dataset.DatasetService;
+import bio.terra.service.resourcemanagement.ResourceService;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.UUID;
+
+public class DeleteDatasetDeleteStorageAccountsStep implements Step {
+    private static Logger logger = LoggerFactory.getLogger(DeleteDatasetDeleteStorageAccountsStep.class);
+    private final ResourceService resourceService;
+    private final DatasetService datasetService;
+    private final UUID datasetId;
+
+    public DeleteDatasetDeleteStorageAccountsStep(ResourceService resourceService,
+                                                  DatasetService datasetService,
+                                                  UUID datasetId) {
+        this.resourceService = resourceService;
+        this.datasetService = datasetService;
+        this.datasetId = datasetId;
+    }
+
+    @Override
+    public StepResult doStep(FlightContext context) throws InterruptedException {
+        Dataset dataset = datasetService.retrieve(datasetId);
+        boolean isAzureDataset = dataset.getStorage().stream()
+            .anyMatch(s -> s.getCloudPlatform() == CloudPlatform.AZURE);
+        if (isAzureDataset) {
+            logger.info("Deleting a storage account for Azure backed dataset");
```

Comment:
Is it typical to `info` log in our steps?

Reply:
Yeah, we do some of that.  It certainly happens within the dao methods.  Makes figuring out what's going on with flights easier